modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase

```diff
@@ -38,13 +38,27 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6085. UseSubscriptions"
-	@ignore = "true"
 	@priority = "3"
 	test UseSubscriptions {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6085 UseSubscriptions
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
 
+		CPCommerceCatalog.newProductsEntry(
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Subscription");
+
+		Click(locator1 = "CommerceEntry#PAYMENT_SUBSCRIPTION_TOGGLE");
+
+		Click(locator1 = "CommerceEntry#DELIVERY_SUBSCRIPTION_TOGGLE");
```

Comment:
For the locator i think is possible to use the macro Check.checkToggleSwitch with:
key_toggleSwitchLabel = "Delivery Subscription"
locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH"